### PR TITLE
Prestonc/patch

### PR DESCRIPTION
**Pull request by prestoncarman**
Testing out a new review tool. What do you think?

### CODE REVIEW

File: src/site/apt/cluster_installation.apt [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+---
+$ps -ef|grep ${USER}|grep java|grep 'Dapp.name=vxquerync'
+---
+
+
+ * <<Check process status for hyracks process>>
+
+---
+$ps -ef|grep ${USER}|grep java|grep 'hyracks'
+---
```
Comment (westmann) [inline src/site/apt/cluster_installation.apt:149]: Is the documentation really part of the patch?

File: vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/RewriteRuleset.java [hunk 2]
```diff
@@ -269,8 +271,8 @@
         prepareForJobGenRewrites.add(new IsolateHyracksOperatorsRule(
                 HeuristicOptimizer.hyraxOperatorsBelowWhichJobGenIsDisabled));
         prepareForJobGenRewrites.add(new ExtractCommonOperatorsRule());
-//        prepareForJobGenRewrites.add(new VXQueryExtractCommonOperatorsRule());
-//        prepareForJobGenRewrites.add(new DelayMaterializationForJoinProbeRule());
+        //        prepareForJobGenRewrites.add(new VXQueryExtractCommonOperatorsRule());
+        //        prepareForJobGenRewrites.add(new DelayMaterializationForJoinProbeRule());
```
Comment (westmann) [inline vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/RewriteRuleset.java]: tabs ...

Comment (westmann) [inline vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/RewriteRuleset.java]: tabs?

File: vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/rules/AbstractUsedVariablesProcessingRule.java [hunk 3]
```diff
@@ -51,8 +52,11 @@ public boolean rewritePost(Mutable<ILogicalOperator> opRef, IOptimizationContext
 
     @Override
     public boolean rewritePre(Mutable<ILogicalOperator> opRef, IOptimizationContext context) throws AlgebricksException {
-        if (hasRun) {
+        opRef.getValue().getOperatorTag();
```
Comment (westmann) [inline vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/rules/AbstractUsedVariablesProcessingRule.java]: remove?

File: vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/step/DescendantOrSelfPathStepUnnesting.java [hunk 4]
```diff
@@ -33,10 +33,11 @@
 
 public class DescendantOrSelfPathStepUnnesting extends AbstractForwardAxisPathStep {
     private boolean testSelf;
+    private boolean printSelf;
```
Comment (westmann) [inline vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/step/DescendantOrSelfPathStepUnnesting.java]: call this "returnSelf"

File: vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/step/DescendantOrSelfPathStepUnnesting.java [hunk 5]
```diff
@@ -33,10 +33,11 @@
...
     private int indexSeqArgs;
     private int seqArgsLength;
     private List<Integer> indexSequence = new ArrayList<Integer>();
-    private List<Boolean> checkSelf = new ArrayList<Boolean>();
+    private List<Integer> printedSequence = new ArrayList<Integer>();
```
Comment (westmann) [inline vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/step/DescendantOrSelfPathStepUnnesting.java]: "returnedSequence"

**Comment by prestoncarman** [comment]
Added to master with suggested changes.

**Pull request closed without merging**